content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/mpcie-4g-modem/content.md

```diff
@@ -358,6 +412,54 @@ udhcpc -q -f -n -i wwan0
 
 ![PRO 4G GNSS Module - Dynamic IP Configuration](assets/portentaMAXcarrier_mpcie_dynamic.png)
 
+### Modem Power Management
+
+The modems might become unresponsive, so it is recommended that power can be controlled through software to allow modem rebooting when necessary. The **gpioset** command should include a jumper and a 20 second delay for proper initialization.
+
+```bash
+gpioset gpiochip5 5=1 #PCIE 3V3 BUCK EN (stm32h7 PE10)
+```
+
+This is applicable to both USB based and QMI based modems. After powering on the modem, allow **20 seconds** for the modem to initialize properly:
+
+```bash
+sleep 20
+```
+
+This ensures the modem powers up correctly and becomes available for network operations.
+
+### Docker Container Considerations
+
+In a Docker environment, it is often useful to disable **ModemManager** to avoid conflicts and instead control the modem using **qmicli** for QMI-based modems or use **nmcli/mmcli** for USB-based modems:
+
+```bash
+sudo systemctl stop ModemManager
+```
+
+For modem power management, the connection between the **PCIE Enable (GPIO5)** pin and **VCC (3V3)** pin is required for a proper power setup. This bridge ensures proper hardware functionality when controlling the modem power. Moreover, you should manage power through software to allow the modem to be rebooted in case it becomes unresponsive.
+
+Ensure that the Docker container has access to the GPIO device files by passing them into the container:
+
+```bash
+docker run --device /dev/gpiochip5 <docker-image>
```

Comment:
which docker image should I put here? any way to explain it? thanks!

Reply:
It would be with the name of the Docker image you want to run with access to the GPIO device files. Updated for clarification